### DISCUSSION TITLE
Roofing contractor

### DISCUSSION THREAD

**Discussion by ellielo** [NAIST-SE/AttractingContributors#1204] [General]
My insurance (home) company asked me to have the roof inspected and provide them with a report and make any repairs as needed.

Could someone suggest a roofing contractor that you may have experience with. Thanks in advance

**Comment by MirkosMangle** [comment]
Got quotes from Shanco and D&M for my dad's house in 2022. Went with D&M Roofing, very satisfied with roof and skylight replacement. I hope plywood sheet pricing has dropped since. Pretty much peak when we did it. Also, the fall/winter is the best time since business is slow.

**Comment by hernessteve** [comment]
Were you able to find a good contractor? I've heard some positive things about roofing companies in Gulf Breeze [https://emeraldcoastroofingexperts.com/gulf-breeze-fl/](https://emeraldcoastroofingexperts.com/gulf-breeze-fl/) —they seem to have a solid reputation. I’m not sure how far those guys are from you, but it might be worth looking into if they serve your area. A reliable roofer can make a huge difference, especially when dealing with something as crucial as your roof. If you haven’t found anyone yet, it might be worth giving them a call to see if they can help
